### PR TITLE
Update jq.yml

### PR DESCRIPTION
**Pull request by Freed-Wu**
I rewrite jq.yml to make it better :smile:

### CODE REVIEW

**Comment by Freed-Wu** [comment]
@lambdalisue

File: .github/workflows/jq.yml [hunk 1]
```diff
@@ -3,10 +3,14 @@
...
-      - '**.json'
+      - ${{ env.paths }}
   pull_request:
     paths:
-      - '**.json'
+      - ${{ env.paths }}
+  workflow_dispatch:
+
+env:
+  paths: assets/json/*.json
```
Comment (lambdalisue) [inline .github/workflows/jq.yml]: `env` is not for defining a variable but for defining environment variable so it should not be used like this.

**Review by lambdalisue — CHANGES_REQUESTED** [review]

File: .github/workflows/jq.yml (hunk 1 above)
Comment (Freed-Wu) [reply .github/workflows/jq.yml]: However, github action don't support yaml's anchor, I don't know how to realize this function except env?

Comment (lambdalisue) [reply .github/workflows/jq.yml]: Yes. So we don't do that.

**Comment by Freed-Wu** [comment]
Fixed.

**Review by lambdalisue — APPROVED** [review]
LGTM

**Pull request merged**